addons/thtracker/thtracker.lua

```diff
@@ -51,7 +52,9 @@ defaults.bg.blue = 30
 
 settings = config.load(defaults)
 
-th = texts.new('No current mob', settings)
+th = texts.new('${th_string}', settings)
```

Comment:
If a text box contains no actual variable splicing (as in, the entire text box is just one variable), it performs better to not use variables and just update the entire text with `th:text(new_text)`. Not a real issue, just mentioning it for future development.